### ISSUE TITLE
Years Active

### ISSUE THREAD

**Issue by tomatizzle** [ynnadkrap/balldontlie#18]
Hi, would it be possible in the `Get All Players` response get the years that a player was active?

e.g. `active: 1979-1988` or `active: 2018-present`

**Comment by cbatson** [comment]
Agreed. For season averages, right now I have to issue a query for every sequential year in order to discover which seasons have season averages -- which quickly runs into the query throttle.
